admin/controllers/service-manager.js

```diff
@@ -284,6 +283,74 @@ const controller = {
     return res.redirect('/service-manager/check-windows')
   },
 
+  /**
+   * Edit check window form.
+   * @param req
+   * @param res
+   * @param next
+   * @returns {Promise.<void>}
+   */
+
+  getCheckWindowEditForm: async (req, res, next) => {
+    req.breadcrumbs('Manage check windows', '/service-manager/check-windows')
+    res.locals.pageTitle = 'Edit check window'
+    let checkWindowData
+    try {
+      checkWindowData = await checkWindowService.getCheckWindowEditForm(req.params.id)
+    } catch (error) {
+      return next()
```

Comment:
Can we do `next(error)` to pass the error to see it in the logs too? Or is this intended, to show the listing page for non-existing check windows ids?

Reply:
@danwilliams it was mistakenly forgotten and now updated